### ISSUE TITLE
Speakers not muted when headphones are plugged in

### ISSUE THREAD

**Issue by rsd** [shibajee/realtek-uad-sonicstudio-mod#2]
I have a problem that, when I plug my headphones to the headphone jack, the speakers keeps outputting sound.

I traced down the problem to an option that is not displayed in the Audio Console:
"Make internal and external output devices playback two different audio streams simultaneously"

This is the Realtek Audio Console from Gigabyte that has this option:
![gigabyte](https://github.com/shibajee/realtek-uad-sonicstudio-mod/assets/197183/f8936f83-425d-4c62-8071-e548a58927f7)

So, I guess this option is by default on.

Ideally would be nice to have a Realtek Audio Controller with this option or to set in the driver off by default.

Maybe there is some registry key or ini file that I can tweak to solve this?

**Comment by shibajee** [comment]
> I have a problem that, when I plug my headphones to the headphone jack, the speakers keeps outputting sound.
> 
> I traced down the problem to an option that is not displayed in the Audio Console: "Make internal and external output devices playback two different audio streams simultaneously"
> 
> This is the Realtek Audio Console from Gigabyte that has this option: ![gigabyte](https://private-user-images.githubusercontent.com/197183/298286290-f8936f83-425d-4c62-8071-e548a58927f7.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDY2MjU0NTEsIm5iZiI6MTcwNjYyNTE1MSwicGF0aCI6Ii8xOTcxODMvMjk4Mjg2MjkwLWY4OTM2ZjgzLTQyNWQtNGM2Mi04MDcxLWU1NDhhNTg5MjdmNy5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMTMwJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDEzMFQxNDMyMzFaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0xZGIwZWMwNzg0NGIzNWJmOTZhNTg5NjY1NTdmNTlhNTMyYTlhMjg1ODFhNjU3N2QzNjA0OTA5NWNlMzdhZTRlJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.G_XQS0KEqFrXklOdQ0NH6GyayXB6ScODbTDDIa1Dh4c)
> 
> So, I guess this option is by default on.
> 
> Ideally would be nice to have a Realtek Audio Controller with this option or to set in the driver off by default.
> 
> Maybe there is some registry key or ini file that I can tweak to solve this?

Yep, this is a common problem. This is mostly due to when realtek shifted from old HDA driver to UAD one. Old HDA driver has a lot of preconfigured options for different motherboard. But UAD is created to unify everything. So, when u switched from HDA to UAD u lost those preconfigured tweaks. I made the minimum changes and keeps thing untouched as much as possible.
